client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -191,6 +192,16 @@ static Request putMapping(PutMappingRequest putMappingRequest) throws IOExceptio
         return request;
     }
 
+    static Request getMappings(GetMappingsRequest getMappingsRequest) throws IOException {
+        Request request = new Request(HttpGet.METHOD_NAME, endpoint(getMappingsRequest.indices(),
+            "_mapping", getMappingsRequest.types()));
```

Comment:
I think that you need to protect both indices and types from NPEs. Unfortunately you can set null from their setter methods. We already do this for other API.